plasmapy/physics/parameters.py

```diff
@@ -524,11 +520,177 @@ def kappa_thermal_speed(T, kappa, particle="e", method="most_probable"):
         raise ValueError("Method {method} not supported in thermal_speed")
 
 
+def particle_params(particle):
+    """Helper function to extract data from particle strings"""
+    try:
+        m = atomic.ion_mass(particle)
+        try:
+            Z = atomic.charge_state(particle)
+        except ValueError:
+            Z = 1
+        Z = abs(Z)
+    except Exception:
+        raise ValueError(f"Invalid particle {particle}")
+    return (m, Z)
+
+
+@utils.check_quantity({
+    'T_e': {'units': units.K, 'can_be_negative': False},
+    'n_e': {'units': units.m**-3, 'can_be_negative': False}
+})
+def electron_ion_collision_rate(T_e, n_e, ion_particle,
+                                coulomb_log=None, V=None):
```

Comment:
If you've already started a new line might as well just separate each argument onto its own line.